src/components/searchbar.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import { fetchMovies } from '../actions/movies';
+
+
+class Searchbar extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            s: ''
+        }
+
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
```

Comment:
You can define those methods in line 19 and 25 like this `handleChange = (e) => {` and binding will not be needed anymore.